android/java/org/chromium/chrome/browser/BraveHelper.java

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser;
+
+import org.chromium.base.CommandLine;
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.browser.ChromeSwitches;
+import org.chromium.chrome.browser.datareduction.DataReductionPromoUtils;
+import org.chromium.chrome.browser.firstrun.FirstRunStatus;
+
+public class BraveHelper {
```

Comment:
Maybe we could use a slightly less generic name for this one? `BraveLauncherHelper` or `BraveActivityHelper`?

Reply:
I wanted to create an actual Helper where we put any common functions. It will be a mess as to me if we create a helper on each action and etc

Reply:
OK, gotcha.